.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# How to contribute
+
+## Code of conduct
+
+We expect all participants to read and adhere to our [code of conduct](/CODE_OF_CONDUCT.md).
+
+## Semantic versioning
+
+Circuit UI follows semantic versioning. In short, this means we use patch versions
+for bugfixes, minor versions for new features, and major versions for
+breaking changes. Our release process is automated using [semantic-release](https://github.com/semantic-release/semantic-release)
+which adheres to the [Angular commit message conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
+
+As a result, our changelog is pretty tidy. You can [read our changelog on GitHub](https://github.com/sumup/circuit-ui/releases).
+
+## Branch organization
+
+We have a couple of special branches that are automatically released whenever
+you merge code into them.
+
+- `master` - A merge to master will release a new version of the library.
+- `canary` - The step before master. This is where we prepare a group of
+  changes for the next release.
+- `beta` - Beta is a branch you can use to test your changes integrated with
+  other branches before it is ready to be PR'd to canary.
+
+## Releases
+
+TODO
```

Comment:
@connor-baer This bit about releases and branch organization could use your help!